cdhweb/events/templates/events/snippets/event_card.html

```diff
@@ -1,9 +1,10 @@
+{% load wagtailcore_tags wagtailimages_tags %}
 {# display a single event in card format #}
 {# FIXME: significant overlap with event_detail, esp. for schema properties #}
 <div class="card event" typeof="schema:Event">
-    <a property="schema:url" href="{{ event.get_absolute_url }}">
-        <div class="event-type">{{ event.event_type }}</div>
-        <div class="image" {% if event.thumb %}style="background-image:url('{{ MEDIA_URL }}{{ event.thumb }}')"{% endif %}></div>
+    <a property="schema:url" href="{{ event.get_url }}">
+        <div class="event-type">{{ event.type }}</div>
+        <div class="image" {% if event.thumbnail %}style="background-image:url('{{ MEDIA_URL }}{{ event.thumb }}')"{% endif %}></div>
```

Comment:
You're using `thumbnail` in the test but outputting `thumb`; maybe this is why images aren't showing up?

Reply:
🤦🏻 good catch, thanks!